frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx

```diff
@@ -125,8 +148,17 @@ export function InsightsTable({ isLegend = true, showTotalCount = false }: Insig
                         hideBreakdown
                         hideIcon
                         useCustomName={!!featureFlags[FEATURE_FLAGS.RENAME_FILTERS]}
+                        innerStyle={
```

Comment:
Nit: I generally would favor a prop that can toggle this stylistic difference or a classname over JSX inline styles passed through. It's ambiguous how inner "inner" really is and which components these styles are supposed to touch without digging into the InsightLabel implementation.

Reply:
Agree with this, and I guess `InsightLabel` should also get the `onClick`, otherwise this happens:

![2021-11-09 10 19 11](https://user-images.githubusercontent.com/53387/140896904-f7c246ed-3dde-40fc-907d-79f03a32b384.gif)

Reply:
Solved